### PR TITLE
Fix printing of PVC information in CRD spec #702

### PR DESCRIPTION
**Pull request by rrati**
Only printing pvc information in the cr spec if the user has provided PVC override information.

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@rrati: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#pullrequestreview-268525757" title="Approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/api/blackduck/v1/types.go [hunk 1]
```diff
@@ -50,26 +50,26 @@ func (b *Blackduck) GenPVC(defaultPVC map[string]string) ([]*components.Persiste
 			pvcMap[claim.Name] = claim
 		}
 
-		for name, defaultSize := range defaultPVC {
```
Comment (msenmurugan) [inline pkg/api/blackduck/v1/types.go]: we should move the method out of v1/types.go. It's not mandatory.

Comment (rrati) [reply pkg/api/blackduck/v1/types.go]: This seems relevant to the BD type, but probably shouldn't be in the api structure.  I'm find moving it if it makes sense, but in a separate PR.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#issuecomment-516533808" title="Approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#pullrequestreview-268525757" title="Approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#issuecomment-516533808" title="Approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#pullrequestreview-268585933" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#pullrequestreview-268525757" title="Approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#" title="Author self-approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,mphammer,msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#issuecomment-516533808" title="Approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#pullrequestreview-268585933" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#pullrequestreview-268525757" title="Approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#" title="Author self-approved">rrati</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/713#issuecomment-516541574" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [jdartigalongue,mphammer,msenmurugan,rrati,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**